lectures/week_07/README.md

```diff
@@ -0,0 +1,108 @@
+```
+echo $RNA_HOME
+```
+
+Since all the environment variables we set up for the RNA-seq workshop start with ‘RNA’ we can easily view them all by combined use of the env and grep commands as shown below. The env command shows all environment variables currently defined and the grep command identifies string matches.
+```
+env | grep RNA
+```
+
+In order to view the contents of this file, you can type:
+```
+less ~/.bashrc
+```
+To exit `less`, type `q`.
+
+### Known Issues/ Discrepancies from RNAbio website
+1. When running the check strandedness tool in the Module 1, RNAseq Data section, the docker run command cannot be run from within your `griffithlab/rnabio:0.0.1` docker session. To run it, we suggest that you open a new terminal window, `cd` into the `rnaseq` directory you created at the beginning of this assignment, and use the following command instead:
+```
+docker run -v $PWD/:/docker_workspace mgibio/checkstrandedness:latest check_strandedness --gtf /docker_workspace/refs/chr22_with_ERCC92_tidy.gtf --transcripts /docker_workspace/refs/chr22_ERCC92_transcripts.clean.fa --reads_1 /docker_workspace/data/HBR_Rep1_ERCC-Mix2_Build37-ErccTranscripts-chr22.read1.fastq.gz --reads_2 /docker_workspace/data/HBR_Rep1_ERCC-Mix2_Build37-ErccTranscripts-chr22.read2.fastq.gz
+```
+This is the same command as what is mentioned in the course webpage, except that instead of mounting (`-v` flag) `/home/ubuntu/workspace/rnaseq` to the docker image- which is where the data was stored for students running through the course on an AMI, you will instead mount whatever your current directory is. Also, this is different from an interactive session where we are able to enter the docker and run commands within it. Instead we are executing our command directly all in that one line of code.
+
+2. In various parts of RNAbio, in order to view HTML files, plots, etc., the tutorial suggests going to a public IPV4 address link in your browser window. That is only needed for the AMI. Since you'll be running everything locally, you can just find the files in your Finder window or File Explorer and open them directly!
+
+3. In Pre-alignment QC, an optional QC analysis is running fastp. This software is not available in your docker, so please skip it (the fastqc and multiqc analysis should still work and can be used for analysis).
+Similarly, you can also skip the adapter trim step as the data provided here does not actually need to be adapter trimmed (however the code is available if you need to do it for your own data)
+ 
+4. `geneBody_coverage.py` in the optional RSeQC section is not correctly in the `PATH`. Use the full path to the python script `/home/ubuntu/.local/bin/geneBody_coverage.py`
+
+
+## Homework Assignments
+1. Finish Module 1 - [Inputs](https://rnabio.org/module-01-inputs/0001/01/01/Intro_to_Inputs/)
+2. Complete Module 2 - [Alignment](https://rnabio.org/module-02-alignment/0002/01/01/Intro_to_Alignment/)
```

Comment:
We have students taking the course for credit and while we're not grading a bunch of questions, we've been asking for some kind of screenshot or answer that proves they made it to the end of the homework. Any suggestions for that?

### For credit students:
Turn in XXXXXX to Jenny as proof of completion

Reply:
I think an IGV screenshot with the merged UHR.bam and HBR.bam could be good?
There's this question towards the end of the alignment visualization section-
What are the options for visualizing splicing or alternative splicing patterns in IGV? Navigate to this location on chromosome 22: ‘chr22:40,363,200-40,367,500’. What splicing event do you see?
An IGV screenshot of that could be good (it should look something like the attached screenshot)
<img width="1331" alt="image" src="https://github.com/user-attachments/assets/e94c66d8-c14e-43f0-902a-71546fd33a0d">
(the text answer to the question is already in rnabio, but the screenshot isn't)

Or we could have them count the number of reads in the merged HBR.bam and UHR.bam files and confirm that they are the sum of the files that were merged. 
HBR.bam = 793963
UHR.bam = 1174955

Reply:
sounds good to me

Reply:
Sweet, adding that to the readme and updating the PR.